### PR TITLE
Add etcd image.

### PR DESCRIPTION
**Pull request by dlorenc**
Signed-off-by: Dan Lorenc <dlorenc@chainguard.dev>

### CODE REVIEW

File: images/etcd/configs/latest.apko.yaml
Comment (amouat) [inline images/etcd/configs/latest.apko.yaml:9]: We shouldn't need to specify glibc.

**Review by amouat — CHANGES_REQUESTED** [review]
Is this something we should worry about? 

https://github.com/etcd-io/etcd/blob/main/Dockerfile-release.amd64#L13

I see the reference is to "alpine", but the FROM is for a distroless image.

Also, there is a _lot_ more in the bitnami image regarding start-up, but most of it seems concerned with mapping env vars to config.

File: images/etcd/configs/latest.apko.yaml
Comment (dlorenc) [reply images/etcd/configs/latest.apko.yaml:9]: Double checked and we do need to keep glibc here - the builds are dynamic on purpose.

**Comment by dlorenc** [comment]
> Is this something we should worry about?
> 
> https://github.com/etcd-io/etcd/blob/main/Dockerfile-release.amd64#L13
> 
> I see the reference is to "alpine", but the FROM is for a distroless image.
> 
> Also, there is a _lot_ more in the bitnami image regarding start-up, but most of it seems concerned with mapping env vars to config.

I think this comment is out of date and they probably meant distroless. But distroless does include an nsswitch.conf now, as does Wolfi.

**Review by jdolitsky — APPROVED** [review]
cannot speak to Adrian's comments, but the repo changes lgtm

**Comment by amouat** [comment]
Yeah, it just needs the glibc entry removed now that it's in wolfi. 

Should we mark this experimental though? etcd is a critical component so we might want to be careful.

**Comment by jdolitsky** [comment]
how to do so if we wanna: https://github.com/chainguard-images/images/blob/main/images/bazel/image.yaml#L1

**Review by amouat — APPROVED** [review]

**Comment by amouat** [comment]
LGTM, but not sure if you're still testing @dlorenc so I'll let you merge.

**Pull request merged**
